### DISCUSSION TITLE
posts/sync-google-drive-linux-clone/

### DISCUSSION THREAD

**Discussion by giscus** [tazihad/tazihad.github.io#2] [Announcements]
# posts/sync-google-drive-linux-clone/

Google Drive doesn’t have any official linux client. Over the years many unofficial method was used to integrate google drive in linux. Mounting as file system or KDE and Gnome’s Google Drive mount. But none of them sync like official client in windows. What we want is two way sync.

https://zihad.com.bd/posts/sync-google-drive-linux-clone/

**Comment by Jwink3101** [comment]
Some comments:

You should remove `--resync`. It should only be used when there is an issue or the *first* sync. Actually, to be fair, you *did* remove it in the systemd script but you should call out the difference for other readers.

---

You said: 

> none of them sync like official client in windows

Unless I am mistaken, the official client doesn't actually sync either anymore. It streams and downloads on demand with the *option* to do a real sync. Something between bisync and mount

---

On the note of mount: you should probably discuss that too. It is also an rclone-provided alternative to first-party clients. It has pros and cons compared to bisync.

---

Many package managers do not keep the most up-to-date rclone versions. rclone is one of the few examples where it is better to install directly. Unless this has changed which is possible.
